lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -3794,6 +3794,11 @@ namespace Js
             library->AddFunctionToLibraryObject(objectConstructor, PropertyIds::defineProperty, &JavascriptObject::EntryInfo::DefineProperty, 3));
         scriptContext->SetBuiltInLibraryFunction(JavascriptObject::EntryInfo::GetOwnPropertyDescriptor.GetOriginalEntryPoint(),
             library->AddFunctionToLibraryObject(objectConstructor, PropertyIds::getOwnPropertyDescriptor, &JavascriptObject::EntryInfo::GetOwnPropertyDescriptor, 2));
+        if (scriptContext->GetConfig()->IsObjectGetOwnPropertyDescriptorsEnabled())
+        {
+            scriptContext->SetBuiltInLibraryFunction(JavascriptObject::EntryInfo::GetOwnPropertyDescriptors.GetOriginalEntryPoint(),
```

Comment:
propertyCount above needs to be incremented.  It is a hint for the number of properties the object is going to have.  I believe we round up to the nearest 4 or 8, but regardless, the number we supply should be exact or at least exactly the maximum possible (with all features enabled)